src/pages/Sample/index.tsx

```diff
@@ -74,6 +174,62 @@ const SamplePage: React.FC = () => {
           </section>
         </div>
       </section>
+
+      {eezData.eezInfoText && (
+        <div className="vf-box vf-box-theme--primary vf-box--easy">
+          <h6 className="vf-box__heading">EEZ Metadata</h6>
+          <p className="vf-box__text">
+            <aside className="vf-article-meta-information">
+              {eezData.eezInfoText && (
+                <div className="vf-meta__details">
+                  <p>
+                    <span
+                      className={`vf-badge vf-badge--${eezData.eezBadgeColor}`}
+                    >
+                      <abbr
+                        title="Exclusive Economic Zone"
+                        className="eez-abbr"
+                      >
+                        EEZ Info
+                      </abbr>
+                    </span>
+                    &nbsp; {eezData.eezInfoText}
+                  </p>
+                </div>
+              )}
+              {eezData.qualifiesForAbsCheck && (
+                <div
+                  dangerouslySetInnerHTML={{
```

Comment:
Could `displayAbsInfo` be refactored into a component, so that it can be conventionally rendered without `dangerouslySetInnerHTML`?

Reply:
@SandyRogers the main reason I resorted to this is that I want to avoid a situation where I'd have to use a ternary statement within the html. Personally I find it to be a bit unwieldy to have chunks of HTML surrounded in a ternary statement. If I change this to use a React component, I would still have to use the ternary statement. 
I thought it might be ok to use dangerouslySetInnerHTML here, since the HTML is coming from code that I control, so there is no real danger of having malicious html/scripts inserted into it by a rouge user.